env/dev/ecs.tf

```diff
@@ -47,7 +47,8 @@ resource "aws_ecs_task_definition" "app" {
     "portMappings": [
       {
         "protocol": "tcp",
-        "containerPort": ${var.container_port}
+        "containerPort": ${var.container_port},
+        "hostPort": ${var.container_port}
```

Comment:
What's hostPort for?

Reply:
As per the AWS docs, the hostPort for a portMapping must either be 0 or equal to the containerPort when the network mode is `awsvpc`. As it so happens, ECS happily makes this transformation atop the incoming container definition.

https://github.com/terraform-providers/terraform-provider-aws/issues/3401